packages/block-editor/src/components/block-list/block.js

```diff
@@ -197,23 +200,40 @@ function BlockListBlock( {
 		);
 	}
 
-	const value = {
-		clientId,
-		rootClientId,
-		isSelected,
-		isFirstMultiSelected,
-		isLastMultiSelected,
-		isPartOfMultiSelection,
-		enableAnimation,
-		index,
-		className: wrapperClassName,
-		isLocked,
-		name,
-		mode,
-		blockTitle: blockType.title,
-		wrapperProps,
-	};
-	const memoizedValue = useMemo( () => value, Object.values( value ) );
+	const memoizedValue = useMemo(
+		() => ( {
+			clientId,
+			rootClientId,
+			isSelected,
+			isFirstMultiSelected,
+			isLastMultiSelected,
+			isPartOfMultiSelection,
+			enableAnimation,
+			index,
+			className: wrapperClassName,
+			isLocked,
+			name,
+			mode,
+			blockTitle: blockType.title,
+			wrapperProps,
+		} ),
+		[
+			clientId,
+			rootClientId,
+			isSelected,
+			isFirstMultiSelected,
+			isLastMultiSelected,
+			isPartOfMultiSelection,
+			enableAnimation,
+			index,
+			wrapperClassName,
+			isLocked,
+			name,
+			mode,
+			blockType.title,
+			wrapperProps,
+		]
+	);
```

Comment:
This is basically the same thing, too bad that we're forced to do that due to the lint rule.

Reply:
Yeah, it's a bummer, we can create a custom hook for this, but I doubt that it's worth it. It doesn't actually hurt to just write it like this anyway, at least it makes the compiler happy 😅

Reply:
Are we forced to use that rule?

Reply:
That's the whole point of this PR though 😅. IMO, The benefits outweighs the inconveniences.

Reply:
If we wanted to, we could use a `/* eslint-disable RULE-NAME */` for this particular use-case, with a comment explaining why.